active_learning/liver_tumor_al/results_uncertainty_analysis.ipynb

```diff
@@ -0,0 +1,420 @@
+{
```

Comment:
<p>May I ask what the "0, 1, 2" in the cell output means? I didn't see any other print or log in the code cell.</p><p> </p><p>Thanks,</p><p>Bin</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1365/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
@binliunls that was just there to make sure that the code was looping over as a debug print statement, the statement of 'print(_mc)' has been removed when the commit was made.

To remove it from output it would need to be re-run from me.

Reply:
Ok, make sense to me. I think @mingxin-zheng is working on this.